### PR TITLE
Feature/typo fix

### PR DESCRIPTION
**Pull request by sillyleo**
- fix typos (style only)
- switch to "trade" tab if connected to a wallet (need review)

### CODE REVIEW

File: src/page/Home/index.tsx [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+    } = User.useContainer()
+
+    useEffect(() => {
+        if (address) {
+            setTabIndex(1)
+        } else {
+            setTabIndex(0)
+        }
+    }, [address])
+
```
Comment (lsps9150414) [inline src/page/Home/index.tsx:24]: @DylanYang0523 Could you help review this part?

File: src/page/Home/index.tsx [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+import { useState, useEffect } from "react"
 import GetStarted from "./component/GetStarted"
 import Position from "./component/Position"
 import TradeComponent from "./component/Trade"
+import { User } from "container/user"
 
 const Home = () => {
     const [tabIndex, setTabIndex] = useState(0)
 
+    // Need review: 偵測到 connect wallet 以後，設定  setTabIndex(1)，才不必每次都看到 get started
```
[the remark this answers is not in the document]
Comment (sillyleo) [reply src/page/Home/index.tsx:11]: No problem

File: src/page/Home/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (sillyleo) [reply src/page/Home/index.tsx:24]: Yes. Maybe we can improve it later, but it's good for me now.

File: src/page/Home/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (lsps9150414) [reply src/page/Home/index.tsx]: updated

**Review by lsps9150414 — APPROVED** [review]

**Pull request merged**